### ISSUE TITLE
The fancy unix script for installing npm is not working

### ISSUE THREAD

**Issue by deostroll** [npm/npm#17554]
Tried to install via the bash [script](https://github.com/npm/npm#fancy-install-unix) on ubuntu. Then my shell complains it can't find npm.

What I think it is doing is tar-ring the downloaded (latest) archive file into  some path in `/tmp` and post install process, in the `~/local` (where node is installed) `npm` is finally symlinked to path in the tmp directory...  

Ps: I followed @isaacs gist on how to go about installing node/npm ([here](https://gist.github.com/isaacs/579814#file-node-and-npm-in-30-seconds-sh))

**Comment by hieudang9** [comment]
Same here.Arch Linux latest
```
[~] : curl -L https://www.npmjs.com/install.sh | sh                                                                            11:36:32
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  6255  100  6255    0     0   8309      0 --:--:-- --:--:-- --:--:--  8306
tar=/usr/bin/tar
version:
tar (GNU tar) 1.29
Copyright (C) 2015 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
install npm@latest
fetching: https://registry.npmjs.org/npm/-/npm-5.0.4.tgz
added 1 package and removed 2 packages in 2.008s
+ npm@5.0.4
added 1 package in 6.891s
It worked
[~] : npm                                                                                                                      11:36:54
fish: Unknown command 'npm'
[~] : nodejs                                                                                                                   11:36:59
fish: Unknown command 'nodejs'
```

**Comment by zkat** [comment]
This should be fixed with the next npm release. It was supposed to be fixed with `npm@5.1.0`, but the commit somehow disappeared when I ran the release. Oops!

The next release might be tomorrow or something. 👍
